google/cloud/aiplatform/initializer.py

```diff
@@ -44,7 +45,6 @@ class _Config:
 
     def __init__(self):
         self._project = None
-        self._experiment = None
```

Comment:
I think we need to keep this field so that the `init` call can reset `run` w/o changing experiment info.

Reply:
This is kept on the _MetadataService class, and the way of doing that would be to just call `set_run`. 
We can discuss further, but I think the set_run way has cleaner semantics